extension/src/data/index.ts

```diff
@@ -58,15 +66,13 @@ export abstract class BaseData<
     this.staticFiles = staticFiles
 
     this.watchFiles()
-
-    this.waitForInitialData()
```

Comment:
[F] Had to rework this because we are now sending multiple notifications per update for experiments.